app/assets/stylesheets/components/_glance-metric.scss

```diff
@@ -13,6 +13,8 @@
 .app-c-glance-metric__heading {
   @include govuk-responsive-margin(3, "top");
   @include govuk-responsive-margin(2, "bottom");
+  min-height: 2.6em;
+
 
   @include media-down(tablet) {
```

Comment:
Non-blocker: instead of `media-down(tablet)` — which is a mixin that we will deprecate — you can use the govuk-frontend version of it, which is `@include govuk-media-query($from: tablet)`; `mobile`, `tablet` and `desktop`) being the breaking points, always going up as the intended approach is to go mobile first. This need quite a bit of refactoring, so it's not a blocking comment, but more for awareness.

Reply:
Fair enough - this stemmed partially from us having no mobile designs when I built the first iteration. I'll bear it in mind for future and switch out as I go when revisiting the CSS, just so we can push on for now.